model/validation.go

```diff
@@ -310,8 +312,9 @@ func ValidateDestinationWeight(dw *proxyconfig.DestinationWeight) (errs error) {
 // ValidateHTTPTimeout validates HTTP Timeout
 func ValidateHTTPTimeout(timeout *proxyconfig.HTTPTimeout) error {
 	if simple := timeout.GetSimpleTimeout(); simple != nil {
-		if simple.TimeoutSeconds < 0 {
-			return fmt.Errorf("timeout_seconds must be in range [0..]")
+		err := validateDuration(simple.Timeout)
+		if err != nil {
+			return fmt.Errorf("timeout must be in range [0..]")
```

Comment:
nice catch, looks like I made the same mistake a few times.  fixed other occurrences as well